randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1435,9 +1451,14 @@ Extra - asset_id: collision_camera_011
   > Door to Save Station West
       Trivial
   > Dock to EMMI Zone Exit Northwest
-      All of the following:
-          Gravity Suit
-          Simple IBJ or Use Spin Boost
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Any of the following:
+                  Speed Booster or Simple IBJ or Use Spin Boost
+                  Flash Shift and Walljump (Beginner)
+                  Morph Ball and Single-wall Wall Jump (Beginner)
```

Comment:
this is one i would consider intermediate since you need more than one morph wall jump for this. Or at least, I'd think you do. A beginner would be something like Energy Recharge Station South